linters/sqlfluff/sqlfluff.test.ts

```diff
@@ -9,7 +9,7 @@ linterCheckTest({ linterName: "sqlfluff", namedTestPrefixes: ["basic_check"] });
 const fmtCallbacks: TestCallback = (driver) => {
   const trunkYamlPath = ".trunk/trunk.yaml";
   const currentContents = driver.readFile(trunkYamlPath);
-  const sqlfluffRegex = /- sqlfluff@((\d.?)+)\n/;
+  const sqlfluffRegex = /- sqlfluff@(.+)\n/;
```

Comment:
codeql says this regex is vulnerable to DDoS, seems the simpler regex is fine anyways.